### PR TITLE
WG Tenacity Buff fix thanks HeartWell for help

### LINKED ISSUE #430: [ZONE][WOTLK] Wintergrasp
**Description**:
Wintergrasp bugs all in one

**Current behaviour**: 
First don't work buff When there are more players in the team than in yours.
Second If all the workshops were in the allience control and wintergrasp to, when won the horde capturing for example 2 workshops, then on the next wintergrasp all the workshops will become in the possession of the alliance 
Anotherone when win horde wintergrasp say "allience bla bla" its not critical but funny :D 
**Expected behaviour**: hard to translate >_< 

**Steps to reproduce the problem**:

1. Go to WG Fight! and you see all bugs from behaviour

**Branch(es)**: master

**AC hash/commit**:  c4ed0375d4f091d2cdc0efbb8ca84bef2d846109

**Operating system**:  Debian 8

**Modules**:  yes transmog by rochet2, arena points announce 

**Other Customizations**: no

### PR DESCRIPTION
**Pull request by Viste**
**Changes proposed:**

-  Fix tenacity buff fix thanks @HeartWell for help

**Target branch(es):** master

**Issues addressed:** partially Closes #430 

**Tests performed:** (Does it build, tested in-game, etc)
Build in game tests

### CODE REVIEW

**Comment by Viste** [comment]
If you just add my PR to your own private repo and don't give feedback i'm stay sad and cry all night like girl 🤣 😃

File: src/game/Battlefield/Zones/BattlefieldWG.cpp [hunk 1]
```diff
@@ -387,11 +389,11 @@ void BattlefieldWG::OnBattleEnd(bool endByTimer)
         
     // Saving data
     for (GameObjectBuilding::const_iterator itr = BuildingsInZone.begin(); itr != BuildingsInZone.end(); ++itr)
-    {
-        (*itr)->Rebuild();
+	{
+		(*itr)->Rebuild();
```
Comment (Rushor) [inline src/game/Battlefield/Zones/BattlefieldWG.cpp]: tabs everywhere

**Comment by starrheld** [comment]
@Viste Will try it out later hopefully, thanks!

**Comment by HeartWell** [comment]
Where thanks @HeartWell?)

**Comment by HeartWell** [comment]
And fix tabs pls.

**Comment by Viste** [comment]
@HeartWell with thanks done  with tabs need go home and sleep in  bus first :D

**Comment by sucofog** [comment]
Fix that tabs please @Viste just add the modiffied lines

http://pastebin.com/hTDKEAJb

**Comment by Viste** [comment]
Tabs fixed ^__^ sorry for so sloooooow but don't have time on that week

**Pull request merged**